### PR TITLE
update paddle tutorial

### PR DESCRIPTION
**Pull request by Yancey0623**

### CODE REVIEW

File: doc/boss_tutorial.md [hunk 1]
```diff
@@ -1,40 +1,45 @@
...
 Baidu scientists and engineers for the purpose of applying deep learning to many
 products at Baidu.
 
-PaddlePaddle Elastic Deep Learning (EDL) is a clustering project which leverages PaddlePaddle training jobs to
-be scalable and fault-tolerant. EDL will greatly boost the parallel distributed training jobs and make good use
-of cluster computing power.
-
-EDL is based on the full fault-tolerant feature of PaddlePaddle, it uses a Kubernetes controller to manage
-the cluster training jobs and an auto-scaler to scale the job's computing resources.
+Fluid is the latest version of PaddlePaddle, which describes the "process" of training
```
Comment (typhoonzero) [inline doc/boss_tutorial.md]: which describes ... => it describes the model for training or inference using the representation of "Program".

File: doc/boss_tutorial.md [hunk 2]
```diff
@@ -1,40 +1,45 @@
...
+    We have some hands-on tutorials after each introduction
+    session so that all the audience can use PaddlePaddle and ask some questions
+    while using PaddlePaddle:
+    - Training models using PaddlePaddle Fluid in a Jupyter Notebook (PaddlePaddle Book).
+    - Laucnh a Distributed Training Job on your laptop.
+    - Launch the EDL training job on a Kubernetes cluster.
+
+- Intended audience
+
+    Peopol who are interested in know the deep learning system architecture.
```
Comment (typhoonzero) [inline doc/boss_tutorial.md]: People

Comment (typhoonzero) [reply doc/boss_tutorial.md]: interested in deep learning system architecture.

File: doc/boss_tutorial.md (hunk 1 above)
Comment (Yancey0623) [reply doc/boss_tutorial.md]: Done.

File: doc/boss_tutorial.md (hunk 2 above)
Comment (Yancey0623) [reply doc/boss_tutorial.md]: Done, thanks.

**Review by typhoonzero — DISMISSED** [review]

File: doc/boss_tutorial.md [hunk 3]
```diff
@@ -1,40 +1,45 @@
...
-  session so that all the audience can use PaddlePaddle and ask some questions
-  while using PaddlePaddle:
-    - Training models using PaddlePaddle Fluid.
-    - Launch EDL training jobs on a Kubernetes cluster.
+
+    We have some hands-on tutorials after each introduction
+    session so that all the audience can use PaddlePaddle and ask some questions
+    while using PaddlePaddle:
+    - Training models using PaddlePaddle Fluid in a Jupyter Notebook (PaddlePaddle Book).
+    - Laucnh a Distributed Training Job on your laptop.
```
Comment (typhoonzero) [inline doc/boss_tutorial.md]: Laucnh =》Launch

Comment (Yancey0623) [reply doc/boss_tutorial.md]: done.

**Review by typhoonzero — APPROVED** [review]

**Pull request merged**